pkg/types/exec_scopes.go

```diff
@@ -44,7 +48,7 @@ func (es *ExecutionScopes) getLocalVariablesMut() (*map[string]interface{}, erro
 	if err != nil {
 		return nil, err
 	}
-	return &locals, err
+	return &locals, nil
```

Comment:
why

Reply:
I think even if it's the same, its more readable to return a nil